co-simulation/bundle/src/assembly/resources/fed/ns3/ns3_installer.sh

```diff
@@ -45,8 +45,8 @@ arg_federate_file=""
 arg_integration_testing=false
 arg_make_parallel=""
 
-required_programs=( python3.6 gcc unzip tar )
-required_libraries=( "libprotobuf-dev (or equal) 3.3.0" "libxml2-dev (or equal)" "libsqlite3-dev (or equal)" )
+required_programs=(python3.6 gcc unzip tar)
```

Comment:
Did you run this file through an auto-formatter or something? Some of these changes are slightly odd, like deleting single spaces, removing semicolons where there were previously used but not strictly necessary. It's not a bad thing, just makes the file a bit harder to review since it's more difficult to identify the meaningful changes when they're mixed in with a bunch of formatting updates.

Reply:
Apologies for the confusion; it appears my VSCode automatically ran a formatting program. The modifications to the ns3_installer.sh file primarily concern the functions `clone_ns3_federate`, `clone_ns3_cv2x`, and `build_ns3`. If you think that is still hard to review, I can revert it and push the change again.